### PR TITLE
Do needed changes to publish on hex

### PR DESCRIPTION
**Pull request by Licenser**
Hi I've published clique as a hex package, this are the changes necessary to make it hex-able with riak3. Since it is your project it would be great if you'd take over the package, for that you can ask hex.pm to change ownership, until then I'll try to keep it up to date with the latest tagged release.

For the meantime I added myself as a secondary maintainer so that people having questions regarding the package don't bother you, please remove that if you take over the package since I don't want to claim your work as mine.

### CODE REVIEW

**Comment by hazen** [comment]
@Licenser Looks like we have a merge conflict.  Did you start with the latest `develop`?

**Comment by Licenser** [comment]
No I started with the latest tag since I didn't wanted to publish the package from develop, but I can re-base if you like

**Comment by Licenser** [comment]
apparently I can't since I suck at gifting -.-

**Comment by Licenser** [comment]
Took me long enough, got it sorted.

File: rebar.config [hunk 1]
```diff
@@ -12,5 +12,5 @@
 ]}.
 
 {deps, [
-  {cuttlefish, ".*", {git, "git://github.com/basho/cuttlefish.git", {tag, "2.0.5"}}}
+  {cuttlefish, "2.0.7"}
```
Comment (hazen) [inline rebar.config]: I only see tags up to 2.0.6 on `cuttlefish.` Is this new?

Comment (Licenser) [inline rebar.config]: I think it's 2.0.6 made to work with rebar3/hex but @tsloughter can answer that question better he made the cuttlefish package

Comment (tsloughter) [inline rebar.config]: What?

**Comment by nickelization** [comment]
Hey @Licenser, sorry this has been sitting ignored, I didn't notice until now that you'd added more commits in August. I definitely appreciate the effort to get this included in hex, and it looks like we're actually using an existing version of cuttlefish now, so this mostly looks good. 

However, the way you're specifying the cuttlefish dep in rebar.config is not familiar to me. Is the `~>` symbol a feature of rebar3, maybe? With rebar2 on my system, if I clone this branch and run "make" I get an error stating "Dependency not available: cuttlefish-~>2.0.8 (undefined)".

Unfortunately Riak is still stuck on rebar2 for now, so we do need to maintain compatibility with that in clique 😞

**Comment by tsloughter** [comment]
That dep `{cuttlefish, "~> 2.0.8"}` is a hex dependency. You are right, it will not work with rebar2.

**Pull request closed without merging**